src/katsdpcontroller/tasks.py

```diff
@@ -736,6 +739,11 @@ async def unhandled_request(self, ctx, req: aiokatcp.core.Message) -> None:
         ctx.reply(aiokatcp.core.Message.OK)
         await ctx.drain()
 
+    def get_command_value(self, value_type: Callable[..., _T], parameter_string: str) -> _T:
```

Comment:
The implementation is quite fragile e.g. it won't work if one uses `--foo=bar` syntax, and it'll crash if the argument is not found. I'm totally fine with that (it's not expected to deal with arbitrary commands), but the methodology should be documented in the docstring.

Reply:
Would `--foo=bar` be a single string instead of subsequent arguments in the list?

Reply:
That would be a single string. But as I say, I think it's easier to just not use that syntax when constructing the command line, as long as this expectation is documented.

I did think about trying to leverage `argparse` to handle the different cases more cleanly, but I think that runs into its own issues because some of the commands are wrappers that use `--` to signal that all following arguments are to be passed to the wrapped program.